pkg/osquery/runtime/runtime_test.go

```diff
@@ -4,38 +4,75 @@ import (
 	"bufio"
 	"bytes"
 	"context"
-	"errors"
 	"fmt"
 	"io"
-	"io/ioutil"
 	"os"
 	"os/exec"
 	"path/filepath"
+	"runtime"
 	"strconv"
 	"syscall"
 	"testing"
 	"time"
 
+	"github.com/kolide/kit/fs"
 	"github.com/kolide/kit/testutil"
+	"github.com/kolide/launcher/pkg/packaging"
 	osquery "github.com/kolide/osquery-go"
+	"github.com/pkg/errors"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+var testOsqueryBinaryDirectory string
+
+// TestMain overrides the default test main function. This allows us to share setup/teardown.
```

Comment:
is there anything that the default test main function does for us? or is it just the `os.Exit(m.Run())` by default?

Reply:
Docs imply that's all it is, and suggest it as the place to hang setup/teardown tasks.